src/lightning_app/cli/cmd_ssh_keys.py

```diff
@@ -0,0 +1,58 @@
+import json
+import random
+import string
+from typing import List, Optional
+
+from lightning_cloud.openapi import V1CreateSSHPublicKeyRequest, V1SSHPublicKey
+from rich.console import Console
+from rich.table import Table
+
+from lightning_app.cli.core import Formatable
+from lightning_app.utilities.network import LightningClient
+
+
+class SSHKeyList(Formatable):
```

Comment:
Let's have these classes protected? We don't expect users to subclass them, or use them independently. At least not at the moment, right?

If you still want to keep it public, I strongly suggest to mark them as experimental, see also #15217

Reply:
sure, let me rename these classes.